src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -182,35 +182,32 @@ function bindCopyButton() {
         return false;
     });
 }
+var prevRow;
 
 function bindCopyEvents() {
-
-    $('#copyTableModal >tbody>tr').on('click', function(e) {
+    $('#copyTableModal > tbody > tr').on('click', function(e){
         e.preventDefault();
-
+        
         var currentRow = $(this);
-        if (currentRow.hasClass('row-selected')) {
-            return;
+        
+        if (typeof prevRow != 'undefined') {
+            $(prevRow).removeClass('row-selected');
+            $($(prevRow).children('td:first')).html('<input type="radio">');
         }
-
+        prevRow = currentRow;
```

Comment:
would be better with a line break after previous code block to help separate logical blocks.

![screen shot 2015-06-18 at 8 12 14 pm](https://cloud.githubusercontent.com/assets/6291947/8230826/657e4bce-15f6-11e5-823f-edca77b84689.png)